lib/user.rb

```diff
@@ -0,0 +1,21 @@
+class User
+  attr_reader :name, :joke
+
+  def initialize(name, joke)
```

Comment:
Keep an eye on the interaction pattern - we should be able to create a User by giving them just a name, instead of a name and a joke.  `User.new('Megan')`